### PR TITLE
Share to meeting in stage view sample nodejs

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Review by Pawan-MSFT — APPROVED** [review]
looks correct, Approving

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/share-to-meeting.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                data-href="<Application-Base-URL>"
+                data-app-id="<Application-ID>"
+                data-entity-name="meeting-test-app"
+				data-button-type="medium"
+				data-icon-px-size="160"
+                data-entity-description="meeting test page to showcase the use of share to meeting functionality"
+            >
+            </div>
+		</div>
+		</>
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/share-to-meeting.jsx:34]: Please check the content alignment.

File: samples/meetings-stage-view/nodejs/ClientApp/src/components/share-to-meeting.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                data-href="<Application-Base-URL>"
+                data-app-id="<Application-ID>"
+                data-entity-name="meeting-test-app"
+				data-button-type="medium"
+				data-icon-px-size="160"
+                data-entity-description="meeting test page to showcase the use of share to meeting functionality"
+            >
+            </div>
+		</div>
+		</>
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/nodejs/ClientApp/src/components/share-to-meeting.jsx:34]: Please check if this alignment is correct.

File: samples/meetings-stage-view/nodejs/ClientApp/src/router/router.js [hunk 3]
```diff
@@ -28,6 +29,7 @@ export const AppRoute = () => {
         <React.Fragment>
             <BrowserRouter>
                 <Routes>
+                    <Route path="/" element={<ShareToMeeting />}/>
                     <Route path="/appInMeeting" element={<AppInMeeting />}/>
                     <Route path="/configure" element={<Configure />}/>
                     <Route path="/home" element={<Home />}/>
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/nodejs/ClientApp/src/router/router.js:35]: Looks like Spacing is not correct, please check once ?
`element={<ShareToMeeting />}`

`element= { <ShareToMeeting /> }`

File: samples/meetings-stage-view/nodejs/Readme.md [hunk 4]
```diff
@@ -73,6 +73,28 @@ For reference please check [Share app content to stage API](https://docs.microso
...
+1) Navigate to [Teams admin portal]("https://admin.teams.microsoft.com/dashboard")
+
+2) Under Teams Apps section, select Manage apps.
+
+3) Search the uploaded app and copy the `App ID`
+![Admin Center](Images/adminCenter.png)
+
+4) Navigate to `samples/samples/meetings-stage-view/nodejs/ClientApp/src/components/app-in-meeting.jsx`
+
+5) On line 41, replace `<<App id>>` with `Id` obtained in step 3.
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/nodejs/Readme.md:88]: Check if line 41 is correct?

File: samples/meetings-stage-view/nodejs/Readme.md [hunk 5]
```diff
@@ -73,6 +73,28 @@ For reference please check [Share app content to stage API](https://docs.microso
...
+
+6) Navigate to `samples/samples/meetings-stage-view/nodejs/ClientApp/src/components/share-to-meeting.jsx`
+
+7) On line 25, replace `<Application-Base-URL>` with your application's base url whrre app is running. E.g. if you are using ngrok it would be something like `https://1234.ngrok.io`.
+
+8) On line 26, replace `<<Application-ID>>` with `Id` obtained in step 3.
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/nodejs/Readme.md:94]: Check this line no as well.
I would suggest, not to specify any line number.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

File: samples/meetings-stage-view/nodejs/Readme.md [hunk 6]
```diff
@@ -73,6 +73,28 @@ For reference please check [Share app content to stage API](https://docs.microso
     ```
     npm start
     ```
+
+## Getting the App id for share to stage deeplink.
+
+1) Navigate to [Teams admin portal]("https://admin.teams.microsoft.com/dashboard")
+
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/nodejs/Readme.md:80]: Please check if you have added further reading section in it.

**Review by ChetanSharma-msft — APPROVED** [review]
Approved with suggestions.

**Review by Wajeed-msft — APPROVED** [review]

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/3779653238
Status:

**Pull request merged**